### PR TITLE
Add PRVD

### PR DESCRIPTION
**Pull request by kthomas**
Add PRVD token

Official Etherscan: https://etherscan.io/token/0x65bb569faadd324a00883fde4c46346cc96d5c0a

### CODE REVIEW

**Comment by kthomas** [comment]
Hi @xrn - is this not sufficient enough to satisfy proof of community activity?
https://etherscan.io/address/0x0a545ab449978f2f9ff74a6f5bf80e6705d8971f

Let me know what else can be done to get this one thru :)

Thanks!

**Comment by kthomas** [comment]
@xrn bump ^^

**Comment by xrn** [comment]
I am looking on activity of token contract - https://etherscan.io/token/0x65BB569FAaDD324a00883FdE4c46346cc96D5c0A

**Comment by kthomas** [comment]
> I am looking on activity of token contract - https://etherscan.io/token/0x65BB569FAaDD324a00883FdE4c46346cc96D5c0A

What are the requirements for "sufficient activity" to get this merged?

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+  "0x65BB569FAaDD324a00883FdE4c46346cc96D5c0A": {
+    "name": "Provide",
+    "logo": "prvd.png",
```
Comment (MRabenda) [inline contract-map.json:4]: SVG is required

Comment (kthomas) [reply contract-map.json]: ty 🙏🏻

**Comment by kthomas** [comment]
Also see https://github.com/MetaMask/contract-metadata/pull/1048 ... if both can be included at this time, you can merge #1048 and close this one 🙏🏻 

Thanks!

**Comment by kthomas** [comment]
@MRabenda closing this PR as we need only merge #1048 👍🏻

**Pull request closed without merging**